libs/designsystem/dropdown/src/dropdown.component.html

```diff
@@ -26,7 +31,7 @@
 </ng-template>
 
 <ng-template #itemWrapperTemplate>
-  <kirby-card>
+  <kirby-card role="listbox" [attr.id]="_listboxId" [attr.aria-labelledby]="_comboboxId + '-label'">
```

Comment:
aria-labelledby should point to the combo-box, which has the label, so I think the '-label' part should be removed such that it matches the id of the combobox
```suggestion
  <kirby-card role="listbox" [attr.id]="_listboxId" [attr.aria-labelledby]="_comboboxId">
```

Reply:
I will delete -label.